eng/scripts/New-DocsMsArtifact.ps1

```diff
@@ -0,0 +1,6 @@
+npm i -g moxygen
+
+# Moxygen will fail if docs/docs.ms does not exist
+mkdir -p docs/docs.ms
+
+moxygen --anchors --output docs/docs.ms/api-docs.md docs/xml
```

Comment:
Will this run per _artifact_ (eg `storage`) or will it run for the entire repo?

Reply:
In C this runs once for the entire repo. For C++ I am planning to have it run once per-package.